flow-jvm/.sdkmanrc

```diff
@@ -0,0 +1 @@
+../.sdkmanrc
```

Comment:
what is this?

Reply:
symlink to parent directory as sdkman (sdkman = makes version management for java, gradle etc easy) does not understand parent directories